fastlane/lib/fastlane/actions/ssh.rb

```diff
@@ -6,7 +6,7 @@ module SharedValues
     end
 
     class SshAction < Action
-      def self.ssh_exec!(ssh, command)
+      def self.ssh_exec!(ssh, command, log)
```

Comment:
Yeah, I guess it's true that it's just being called from its own action, however you never know what people do, basically it's all about https://xkcd.com/1172/